### PR TITLE
feat: hero-updates table

### LINKED ISSUE #64: [Table] DFK Hero Stats
Table that parses hero stats

### PR DESCRIPTION
**Pull request by antonyip**
Closes #64

### CODE REVIEW

File: models/dfk/dfk_hero_updates.sql [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+        ) 
+}}
+
+with
+events as (
+    select
+        *
+    from {{ ref("logs") }}
+    where {{ incremental_load_filter("block_timestamp") }}
+        and topics[0] = '0x7be34da84992130f23438d167b22f7f7a246aaaf2d7e9dd3c988ee1672fe40fc'
```
Comment (chuxinh) [inline models/dfk/dfk_hero_updates.sql:20]: can you add a comment here on what this topic is?

File: models/dfk/dfk_hero_updates.sql [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+),
+final as (
+    select
+        block_timestamp,
+        block_id,
+        log_id,
+        java_hextoint(substr(data,3+64+(0*64),64)) as hero_id,
+        java_hextoint(substr(data,3+64+(1*64),64)) as summoning_info_summonedTime,
```
Comment (chuxinh) [inline models/dfk/dfk_hero_updates.sql:25]: i don't think it matters in snowflake but having the columns all in snake case?

Comment (chuxinh) [inline models/dfk/dfk_hero_updates.sql]: don't think it matters too much in snowflake but maybe have them in snake case?

**Review by chuxinh — APPROVED** [review]
LGTM

File: models/dfk/dfk_hero_updates.sql (hunk 1 above)
Comment (antonyip) [reply models/dfk/dfk_hero_updates.sql:20]: Honestly not sure what event is this as its un-documented.
If you select distinct topics[0] from the above address, its the only topic that doesn't appear in 4byte.directory.

**Pull request merged**